recipe/meta.yaml

```diff
@@ -1,8 +1,14 @@
 {% set name = "google-cloud-bigquery" %}
-{% set version = "2.1.0" %}
-{% set sha256 = "7369e2f81ccddfa62f2af4009dbddf7a1daaef0fc701a4097e8de5508235af74" %}
+{% set version = "2.2.0" %}
+{% set sha256 = "94bcdce4bcf67974445e2c46224dcdab211d91c1dab720fb1dd68884a720a5f1" %}
 {% set dev_url = "https://github.com/googleapis/python-bigquery" %}
 {% set doc_url = "https://googleapis.dev/python/bigquery/latest/index.html" %}
+{% set dependency_bigquery_storage = "google-cloud-bigquery-storage-core >=2.0.0,<3.0.0dev" %}
+{% set dependency_grpcio = "grpcio >=1.32.0,<2.0dev" %}
```

Comment:
I notice `google-api-core-feedstock` uses `pin_subpackage` for this https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#pinning-expressions

We might be able to do something similar instead of defining so many variables.